ui/css/utilities/fonts.scss

```diff
@@ -83,7 +83,7 @@ $font-path: './fonts';
 
 @font-face {
   font-family: 'MMPoly';
-  src: url('#{$font-path}/MMPoly/MM_Poly-Variable.woff2') format('woff2');
+  src: url('#{$font-path}/MMPoly/MM_Poly-Regular.woff2') format('woff2');
```

Comment:
I am not sure, if this is the correct format to use this. cc @georgewrmarshall

Reply:
<img width="1485" alt="Screenshot 2025-06-03 at 12 11 33 AM" src="https://github.com/user-attachments/assets/cef37f27-ad77-4246-87e5-086bb6c058e5" />
The font is broken when i use `Variable` here. That's why I used `Regular`